deisctl/units/deis-kube-kubelet.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Kubernetes Kubelet
+Documentation=https://github.com/GoogleCloudPlatform/kubernetes
+[Service]
+EnvironmentFile=/etc/environment
+ExecStartPre=/usr/bin/wget -P /opt/bin https://storage.googleapis.com/kubernetes-release/release/v0.19.3/bin/linux/amd64/kubelet
+ExecStartPre=/usr/bin/chmod +x /opt/bin/kubelet
+ExecStartPre=/usr/bin/mkdir -p /opt/kubernetes/manifests/
+ExecStart=/bin/bash -c '/opt/bin/kubelet --address=0.0.0.0 --port=10250 --hostname_override=$COREOS_PRIVATE_IPV4 --api_servers=`/usr/bin/etcdctl get /deis/scheduler/k8s/master`:8080 --allow_privileged=true --cluster_dns=10.100.0.10 --cluster_domain=cluster.local --logtostderr=true --config=/opt/kubernetes/manifests/ --healthz_bind_address=0.0.0.0 --healthz_port=10248'
```

Comment:
> I thought we were "containerizing" these services so it's easier to run these locally without meddling with the host filesystem, as well as better packaging, etc

@bacongobbler yes it makes sense but is not ready for that yet.

https://github.com/GoogleCloudPlatform/kubernetes/issues/4869
https://github.com/kelseyhightower/kubeos